bootstrap/frontend/src/main/scala/io/udash/bootstrap/tooltip/TooltipUtils.scala

```diff
@@ -66,16 +86,73 @@ trait TooltipUtils[TooltipType <: Listenable[TooltipType, _]] {
         "content" -> js.ThisFunction.fromFunction1(content),
         "delay" -> js.Dictionary("show" -> delay.show.toMillis, "hide" -> delay.hide.toMillis),
         "html" -> html,
-        "placement" -> scalajs.js.Any.fromFunction2((popover: dom.Element, trigger: dom.Element) => placement(popover, trigger).map(_.name).mkString(" ")),
+        "placement" -> scalajs.js.Any.fromFunction2((popover: dom.Node, trigger: dom.Node) => placement(popover, trigger).map(_.name).mkString(" ")),
         "template" -> template.getOrElse(defaultTemplate),
         "title" -> js.ThisFunction.fromFunction1(title),
         "trigger" -> trigger.map(_.name).mkString(" "),
         "viewport" -> js.Dictionary("selector" -> viewport.selector, "padding" -> viewport.padding)
       )
     )(el)
 
+  /**
+    * Add tooltip/popover to provided element with translated content.
+    * More: <a href="http://getbootstrap.com/javascript/#tooltips">Bootstrap Docs (Tooltip)</a>.
+    * More: <a href="http://getbootstrap.com/javascript/#popovers">Bootstrap Docs (Popover)</a>.
+    *
+    * @param animation Apply a CSS fade transition to the popover.
+    * @param container Appends the popover to a specific element.
+    * @param content   Popover content.
+    * @param delay     Show/hide delay.
+    * @param html      Treat content and title as HTML.
+    * @param placement Tooltip/popover placement.
+    * @param template  Tooltip/popover template.
+    * @param title     Component title.
+    * @param trigger   Triggers to show/hide tooltip.
+    * @param viewport  Keeps the popover within the bounds of this element.
+    * @param el        Node which will own the created tooltip/popover.
+    */
+  def i18n(animation: Boolean = true,
+           container: Option[String] = None,
+           content: (dom.Node) => TranslationKey0 = (_) => TranslationKey.untranslatable(""),
+           delay: Delay = Delay(0 millis, 0 millis),
+           html: Boolean = false,
+           placement: (dom.Node, dom.Node) => Seq[Placement] = defaultPlacement,
+           template: Option[String] = None,
+           title: (dom.Node) => TranslationKey0 = (_) => TranslationKey.untranslatable(""),
+           trigger: Seq[Trigger] = defaultTrigger,
+           viewport: Viewport = Viewport("body", 0))
+          (el: dom.Node)
+          (implicit langProperty: LangProperty, translationProvider: TranslationProvider): TooltipType = {
+
+    val tp = apply(animation, container, _ => "", delay, html, placement, template, _ => "", trigger, viewport)(el)
+
+    var lastContent = ""
+    var lastTitle = ""
+    def updateContent(): Unit = {
```

Comment:
I don't particularly enjoy this style with logic encapsulated in a locally created closure, but there aren't many better ways to do this (unless the `lastContent` and `lastTitle` are useful data for a tooltip user, which they could be - up to you). Can we at least have a test? ;)

Reply:
It's a hack. I'm not proud of this code, but it works. ;)